MobiusCore/Source/EffectHandlers/EffectRouter.swift

```diff
@@ -37,6 +58,10 @@ public struct EffectRouter<Input, Output> {
         return PartialEffectRouter(routes: routes, path: payload)
     }
 
+    /// Convert this `EffectRouter` into `Connectable` which can be attached to a Mobius Loop, or called on its own to handle effects.
+    ///
+    /// Note: Each instance of `Input` must have been linked to exactly one handler. A runtime crash will occur if zero or multiple handlers were found for some
```

Comment:
Maybe this note belongs in the top-level documentation? I guess most people won't be invoking `toConnectable` themselves and so might not see this comment?

Reply:
Most users would be invoking it. I'll move the comment